packages/databricks-vscode/src/bundle/BundleInitWizard.ts

```diff
@@ -188,12 +188,12 @@ export class BundleInitWizard {
             "bundle",
             "init",
             "--output-dir",
-            this.cli.escapePathArgument(parentFolder.fsPath),
+            `"${parentFolder.fsPath}"`,
```

Comment:
I think mac and linux can technically have `"` in the dir names, so escaping the path can still be useful